src/pages/home/report/ReportActionItemSingle.js

```diff
@@ -56,24 +58,33 @@ const ReportActionItemSingle = ({
     const personArray = displayName
         ? [{type: 'TEXT', text: Str.isSMSLogin(login) ? toLocalPhone(displayName) : displayName}]
         : action.person;
+
+    const showUserDetails = (email) => {
+        Navigation.navigate(`${ROUTES.DETAILS}/${email}`);
+    };
+
```

Comment:
NAB: Instead of declaring this function every time we render this, maybe you can take it out from the render logic here and place it after declaring the `const defaultProps = {...};`. Honestly though, I don't think this is really a pattern we follow (from glancing at other files), so don't feel compelled to do this unless you think it also makes sense.

Reply:
Makes sense! Every bit of performance gain matters 😃